README.markdown

```diff
@@ -533,21 +533,50 @@ UIView.animate(withDuration: 1.0, animations: {
 }) { f in
   self.myView.removeFromSuperview()
 }
+
 ```
 For single-expression closures one one line, use implicit returns:
 
 ```swift
 attendeeList.sort { $0 > $1 }
 ```
 
-For single-expression closures with more than one, use explicit returns:
+For single-expression closures with more than one line, use explicit returns:
 
 ```swift
-attendeeList.sort { a, b in 
+attendeeList.map { a, b in 
+  let sum = a + b
+  return sum
+}
+```
+
+For closures with more than one line, use named parameters
+**Preferred**:
+
+```swift
+attendeeList.sort { a, b in
   return a > b
 }
 ```
 
+**Not Preferred**:
+```swift
+attendeeList.sort {
+  return $0 > $1
+}
+```
+
+If possible avoid more than one line if the expression is easy to evaluate
+**Preferred**:
+```swift
+attendeeList.sort { $0 > $1 }
```

Comment:
How about using closures/free funtions? Should we prefer them in a simple cases like this (I mean `sort(by: >)` for example)?

Reply:
Yeah maybe this is a bad example. Don't think it makes a difference in this rule tho